roles/realm_join/tasks/realm_join_ad.yml

```
@@ -0,0 +1,116 @@
+---
+- name: Ensure pexpect to answer password
+  become: yes
+  package: name=pexpect state=latest
```

Writing it out is generally cleaner (repeated below, not going to repeat the comment)
```suggestion
  package:
    name: pexpect
    state: latest
```